tasks/main.yml

```diff
@@ -1,5 +1,20 @@
 - block:
-  - include: django.yml
+  - include: virtualenv.yml
+
+  - name: Verify that we have a valid django_settings_pattern
```

Comment:
Practically the same thing, but `assert` is a bit more fitting for this: http://docs.ansible.com/ansible/assert_module.html